src/routes/families/+page.server.ts

```diff
@@ -0,0 +1,12 @@
+import { getAllEventTypes, transformForEventPanel } from "$lib/server/calendarUtils"
+import type { PageServerLoad } from "./$types";
+
+export const load = (async () => {
+  const allEventTypes: Array<Calendly.EventTypeData> = await getAllEventTypes();
+  const lectureEventData = transformForEventPanel(allEventTypes.filter(eventType => eventType.internalNote.toLowerCase().includes('lecture')));
```

Comment:
I think we should make them hashtags, that way they can add additional internal notes and the concept should be familiar to them

Reply:
i like that, i can do that!